UofT-DSI | Sampling - Assignment 2

movcha opened the pull request:
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
Designing a survey to study the relationship between age and music taste.
## What did you learn from the changes you have made?
I learned how to structure a survey, select a sampling strategy, and formulate questions that target specific research objectives.
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
I considered using cluster sampling instead of stratified sampling to simplify the process.
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
The main challenge was ensuring the survey questions were clear and unbiased. I overcame this by reviewing and revising the questions multiple times.
## How were these changes tested?
I conducted a pilot survey with my family to test the clarity and effectiveness of the questions.
## A reference to a related issue in your repository (if applicable)
N/A
## Checklist
- [X] I can confirm that my changes are working as intended

movcha committed:
Update a2_questionnaire_design_part_a.md

KasraVakiloroayaei commented:
A2: INCOMPLETE 
Please specify the sampling strategy (SRS/STRATA/CLUSTER, etc..) and justify it.

movcha committed:
specifed the sampling strategy

Stratified sampling strategy should be used. The intention is to ensure that all age groups are represented in the survey, which is crucial for studying how music taste varies with age and increase the accuracy of the results by reducing sampling error within each age stratum.
Using stratified sampling, we can draw more reliable and generalizable conclusions about the relationship between age and music taste.

movcha commented:
@Melokazz

Specifed the sampling strategy, added text:

Stratified sampling strategy should be used. The intention is to ensure that all age groups are represented in the survey, which is crucial for studying how music taste varies with age and increase the accuracy of the results by reducing sampling error within each age stratum.
Using stratified sampling, we can draw more reliable and generalizable conclusions about the relationship between age and music taste.

KasraVakiloroayaei commented:
A2: PASS 
Great work!
